beets/ui/__init__.py

```diff
@@ -1858,6 +1858,16 @@ def main(args=None):
     """Run the main command-line interface for beets. Includes top-level
     exception handlers that print friendly error messages.
     """
+    if "AppData\\Local\\Microsoft\\WindowsApps" in sys.exec_prefix:
+        log.error(
+            "error: beets is unable to use the Microsoft Store version of "
+            "Python. Please install Python from python.org."
+        )
+        log.error(
+            "error: More details can be found here "
+            "https://beets.readthedocs.io/en/stable/guides/main.html"
```

Comment:
Consider using a single `log.error` call:

```suggestion
            "error: beets is unable to use the Microsoft Store version of "
            "Python. Please install Python from python.org. See "
            "https://beets.readthedocs.io/en/stable/guides/main.html."
```

Reply:
Updated